packages/@react-spectrum/s2/src/Fonts.tsx

```diff
@@ -0,0 +1,98 @@
+import ReactDOM from 'react-dom';
+import {ReactNode, version as ReactVersion, useEffect, useMemo, useRef} from 'react';
+import {useLocale} from 'react-aria';
+import './fonts.css';
```

Comment:
is there a use-case to not include these `@font-face` rules? they are only loaded when actually used. If we want an opt-out, it'll be more complicated and won't be possible to include in Provider.